charts/connect/templates/operator-deployment.yaml

```diff
@@ -68,17 +68,26 @@ spec:
                   fieldPath: metadata.name
             - name: OPERATOR_NAME
               value: "{{ .Values.operator.applicationName }}"
-            - name: OP_CONNECT_HOST
-              value: "{{- include "onepassword-connect.url" . }}"
             - name: POLLING_INTERVAL
               value: "{{ .Values.operator.pollingInterval }}"
+            - name: AUTO_RESTART
+              value: "{{ .Values.operator.autoRestart }}"
+            {{- if .Values.operator.serviceAccountToken.value }}
+            - name: OP_SERVICE_ACCOUNT_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.operator.serviceAccountToken.name }}
+                  key: {{ .Values.operator.serviceAccountToken.key }}
+            {{- end }}
+            {{- if .Values.operator.token.value }}
```

Comment:
Shouldn't this also check for the value `.Values.operator.token.name`? My setup doesn't configure the token value via this chart but provides it via a separately provisioned secret. This change seems to break this functionality.